### PR TITLE
Added text and links to dryville_story.md

### PR DESCRIPTION
**Pull request by elmeraa**
Following the gitflow issues instructions, I added a link for getting water to your home (issue #5) and provided more information about Dryville's first water works with hyperlinks (issue #6)

### CODE REVIEW

File: dryville_story.md [hunk 1]
```diff
@@ -15,3 +15,14 @@ So let's say you made the big move. You and some friends have found your (desola
...
 
 In your free time go ahead and start digging a hole in the ground for a [well](https://www.usgs.gov/special-topic/water-science-school/science/groundwater-wells). If you dig deep enough you might hit the [water table](https://www.usgs.gov/special-topic/water-science-school/science/aquifers-and-groundwater), where there will be standing water. You'll be able to lower a bucket down to get the available [groundwater](https://www.usgs.gov/special-topic/water-science-school/science/groundwater-what-groundwater). Things are starting out great -- you've already started using surface water (the creek and pond), ground water (your well), and you've even created your own water-storage system (the reservoir). So you now have a reliable source for your water needs. Time to relax? Not yet.
+
+## [Getting Water to Your Homes](https://www.usgs.gov/special-topics/water-science-school/science/water-density)
```
Comment (jesse-ross) [inline dryville_story.md:19]: I see what you've done here - added the section header and an appropriate link for "over 8 pounds a gallon", like #5 is requesting. It makes sense and the issue isn't very well worded. What we're actually looking for here is for your page to mimic the content [here](https://www.usgs.gov/special-topics/water-science-school/science/story-water-dryville) for the "Getting Water to Your Homes" section. This should include appropriate hyperlinks for "over 8 pounds a gallon" and any other links in the original text. Would you mind updating, please?

**Review by jesse-ross — CHANGES_REQUESTED** [review]
Thanks Elmera; the "Dryville's First Water Works" section looks perfect! I would like some updates to the prior section, though - please see my comment specific to that section.

**Comment by elmeraa** [comment]
pushed an updated dryville_story.md resolving Jesse's notes on "Getting Water to Your Homes" section. I added appropriate hyperlinks when needed, resolving issue #5.

**Review by jesse-ross — APPROVED** [review]

**Comment by jesse-ross** [comment]
Looks great; merge away!

**Pull request merged**